jest.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   moduleNameMapper: {
     '\\.(css|scss|sass|less)$': 'identity-obj-proxy',
     '^@components(.*)$': '<rootDir>/src/components$1',
+    '^@constants(.*)$': '<rootDir>/src/constants$1',
```

Comment:
the `develop` branch already contains this

Reply:
ok it was throwing error during my tests so I added there, its there so i will remove it then